_redirects

```diff
@@ -478,3 +478,11 @@ docs/contributing/documenting-a-new-adapter /docs/contributing/adapter-developme
 /guides/getting-started/learning-more/getting-started-dbt-core /docs/develop/getting-started/getting-started-dbt-core 301
 /guides/getting-started/learning-more/refactoring-legacy-sql /docs/develop/getting-started/learning-more/refactoring-legacy-sql 301
 /guides/getting-started/learning-more/using-jinja /docs/develop/getting-started/using-jinja 301
+
+
+/docs/dbt-cloud/cloud-ide/the-dbt-ide  /docs/develop/dbt-cloud-features 301!
```

Comment:
oh! there was development today. the "Develop with dbt" leftbar has been updated to "Get started with dbt". you can see the merged PR [here](https://github.com/dbt-labs/docs.getdbt.com/pull/2186) . because of this, we'll need to update these `/develop/` redirects and move the MD source files to the new directory in the repo. many apologies for this late change!